csharp-basic-topics/ValidateNumberIsPositiveOrNegative/ValidateNumberIsPositiveOrNegativeTests/ValidateNumberTest.cs

```diff
@@ -0,0 +1,84 @@
+using ValidateNumberIsPositiveOrNegative;
+
+namespace ValidateNumberIsPositiveOrNegativeTests
+{
+    [TestFixture]
+    public class ValidateNumberTest
+    {
+        [TestCase(60, 1)]
+        [TestCase(-30, -1)]
+        [TestCase(0, 0)]
+        public void GivenNumber_WhenUsingConditionalMethod_ThenPrintResult(int number, int expectedOutput)
```

Comment:
Please modify the last part in the name of all the test method to "ThenVerifyResult" since we are not printing the result now.

Reply:
Updated test method names in all testcases.